contract/deploy.js

```diff
@@ -10,7 +10,7 @@ import { E } from '@agoric/eventual-send';
 
 /**
```

Comment:
Is there a reason we can't somehow import these types from the SDK instead of redefining them?

Reply:
Issue here: https://github.com/Agoric/agoric-sdk/issues/3154